app/views/public/hours/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div id="wrapper">
+    <div id="sidebar-wrapper">
+        <ul class="sidebar-nav">
+            <% @presenter.resources_for_list.each do |resource| %>
+                <% resource.each do |item| %>
+                  <% name = @presenter.get_resource_name({id: item.id, type: resource.name.downcase}) %>
+                  <li><%= link_to name, hours_path(:id => item.id, :type => resource.name.downcase) %></li>
+                <% end %>
+            <% end %>
+        </ul>
+    </div>
+    <div id="page-content-wrapper">
+        <div class="page-content">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12">
```

Comment:
Why the extraneous markup?  We aren't using bootstrap and it doesn't need to be there.